src/Illuminate/Contracts/Auth/Authenticatable.php

```diff
@@ -46,4 +46,11 @@ public function setRememberToken($value);
      * @return string
      */
     public function getRememberTokenName();
+
+    /**
+     * Get the column name for the "password".
+     *
+     * @return string
+     */
+    public function getPasswordName();
```

Comment:
Adding a method to an interface is a breaking change so this PR should target `master`.